src/traces/sunburst/layout_attributes.js

```diff
@@ -35,5 +35,15 @@ module.exports = {
             'Colors provided in the trace, using `marker.colors`, are never',
             'extended.'
         ].join(' ')
+    },
+    sort: {
+      valType: 'boolean',
+      dflt: true,
+      role: 'style',
+      editType: 'calc',
+      description: [
+          'sort on values.',
+          'default true'
+      ].join(' ')
```

Comment:
Thanks very much for the PR.
Instead of adding this new attribute to the layout, I suggest you add it to the trace similar to `pie` .
That way one has more control over which `sunburst` should be sorted or not.
For that you can simply start by adding 
```js
sort: pieAttrs.sort,
```
to
https://github.com/plotly/plotly.js/blob/master/src/traces/sunburst/attributes.js
and then `coerce` it in 
https://github.com/plotly/plotly.js/blob/master/src/traces/sunburst/defaults.js

Reply:
Makes sense, thanks.